base/base/cgroupsv2.cpp

```diff
@@ -23,18 +23,19 @@ bool cgroupsV2MemoryControllerEnabled()
 {
 #if defined(OS_LINUX)
     chassert(cgroupsV2Enabled());
-    /// According to https://docs.kernel.org/admin-guide/cgroup-v2.html:
-    /// - file 'cgroup.controllers' defines which controllers *can* be enabled
-    /// - file 'cgroup.subtree_control' defines which controllers *are* enabled
-    /// Caveat: nested groups may disable controllers. For simplicity, check only the top-level group.
-    std::ifstream subtree_control_file(default_cgroups_mount / "cgroup.subtree_control");
-    if (!subtree_control_file.is_open())
+    /// According to https://docs.kernel.org/admin-guide/cgroup-v2.html, file "cgroup.controllers" defines
+    /// which controllers are enabled for the current cgroup. Check the bottom-most nested "cgroup.controllers"
+    /// file.
```

Comment:
https://docs.kernel.org/admin-guide/cgroup-v2.html says differently: `cgroup.controllers` is for available controllers.
It is worth extending the comment.